packages/dependency-extraction-webpack-plugin/index.js

```diff
@@ -15,6 +16,7 @@ class DependencyExtractionWebpackPlugin {
 		this.options = Object.assign(
 			{
 				injectPolyfill: false,
+				outputFormat: 'json',
```

Comment:
If we're already making breaking changes, should the default be `php`? Our primary consumer for these asset files is a PHP application, isn't it?

Should we sanitize allowed `outputFormat` values? We could throw an error early to surface bad values.

Reply:
Yes, I'm leaning towards making `php` the default format 👍

Reply:
199b51a - done